package.json

```diff
@@ -15,5 +15,9 @@
   "keywords": [
     "lemon"
   ],
-  "author": "AlmostGreatBand"
+  "author": "AlmostGreatBand",
+  "dependencies": {
+    "eslint": "^7.11.0",
+    "vue": "^2.6.12"
```

Comment:
Vue on server side?)))

Also, ESLint will be added by #8 when we merge it to master.